apps/frontend/src/hooks/aave/useAaveUserReservesData.tsx

```diff
@@ -92,6 +90,7 @@ export const useAaveUserReservesData = (): {
   useEffect(() => {
     setLoading(true);
     setProcessedBlock(undefined);
+    setProcessedBlock(-1);
```

Comment:
Setting the state to undefined and then to -1 looks weird, is it ok?

Reply:
Totally!! Nice catch! Thanks!